backend/src/zango/cli/start_project.py

```diff
@@ -228,19 +228,32 @@ def start_project(
     # Creating Public Tenant
     create_public_tenant(platform_domain_url=platform_domain_url)
 
-    # Prompting default platform user details
-    if not platform_username:
-        click.echo("Please enter platform user email")
-        platform_username = click.prompt("Email")
-    if not platform_user_password:
-        platform_user_password = click.prompt(
-            "Password", hide_input=True, confirmation_prompt=True
+     # Prompting default platform user details
+    while True:
+        temp_username = platform_username
+        platform_username = None
+        platform_user_password = None
+
+        if not platform_username:
+            click.echo("Please enter platform user email")
+            platform_username = click.prompt("Email")
+        else:
+            click.echo("Please enter platform user email\nPrevious Attempt: " + temp_username)
```

Comment:
ideally, this will never go to this `else` block right? because we are making `platform_username` to `None` at the start of the loop.

Reply:
That's why I saved it to the temp_username I just thought it would be good to show if they made an error or if it was in the password for more feedback I can change it back

Reply:
I just realized the problem sorry I was working on this after work and I should have double-checked! I have it checking for temp and then running if it's filled 

#` 
while True:
        temp_username = platform_username
        platform_username = None
        platform_user_password = None

        if not temp_username:
            click.echo("Please enter platform user email")
            platform_username = click.prompt("Email")
        else:
            click.echo("Please enter platform user email\nPrevious Attempt: " + temp_username)
            platform_username = click.prompt("Email")
        if not platform_user_password:
            platform_user_password = click.prompt(
                "Password", hide_input=True, confirmation_prompt=True
            )

        user_creation_result = create_platform_user(
            platform_username, platform_user_password
        )
        if user_creation_result["success"]:
            break
        else:
            user_creation_result["message"]
            click.echo(user_creation_result["message"])
            retry = click.prompt("Do you want to try again? (yes/no)", default="yes")
            if retry.lower() != "yes":
                raise click.ClickException("User creation aborted by the user.")
`

Reply:
No worries,

There is one another issue in this, if platform_username is passed as `--platform_username` arg in this command then still it's prompting for the username